package-lock.json

```diff
@@ -1,8 +1,5709 @@
 {
   "name": "simple-icons",
   "version": "5.8.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Please revert this back to a `package-lock.json` in the `"lockfileVersion": 1` format. For a (ongoing) related discussion see #6179.

The easiest way is probably to just copy the `package-lock.json` from the `develop` branch and commit the changes to this branch (*). If you need help with this feel free to ask for more guidance.

---

(*): It doesn't matter if this change is in the commit history as we always squash and merge contributions :slightly_smiling_face: